src/Data.js

```diff
@@ -0,0 +1,31 @@
+let shopItemsData = [
```

Comment:
```suggestion
const shopItemsData = [
```
You should use `const`  instead of `let` for declaring your shopItemsData since it is not reassigned or modified later if you understand me. This will help prevent accidental mutations and make your code more predictable.

Reply:
Alright, I'll make the necessary correction then re-push. Thanks bro I appreciate